package.json

```diff
@@ -48,21 +48,21 @@
     "build-storybook": "storybook build"
   },
   "devDependencies": {
-    "@storybook/addon-essentials": "8.0.0-beta.2",
-    "@storybook/addon-interactions": "8.0.0-beta.2",
-    "@storybook/addon-links": "8.0.0-beta.2",
-    "@storybook/blocks": "8.0.0-beta.2",
-    "@storybook/components": "8.0.0-beta.2",
-    "@storybook/core-events": "8.0.0-beta.2",
-    "@storybook/manager": "8.0.0-beta.2",
-    "@storybook/manager-api": "8.0.0-beta.2",
-    "@storybook/preview": "8.0.0-beta.2",
-    "@storybook/preview-api": "8.0.0-beta.2",
-    "@storybook/react": "8.0.0-beta.2",
-    "@storybook/react-vite": "8.0.0-beta.2",
+    "@storybook/addon-essentials": "8.0.0",
+    "@storybook/addon-interactions": "8.0.0",
+    "@storybook/addon-links": "8.0.0",
+    "@storybook/blocks": "8.0.0",
+    "@storybook/components": "8.0.0",
+    "@storybook/core-events": "8.0.0",
+    "@storybook/manager": "8.0.0",
+    "@storybook/manager-api": "8.0.0",
+    "@storybook/preview": "8.0.0",
+    "@storybook/preview-api": "8.0.0",
+    "@storybook/react": "8.0.0",
+    "@storybook/react-vite": "8.0.0",
     "@storybook/testing-library": "^0.2.2",
```

Comment:
we should get rid of this and use @storybook/test instead. Even better, if this package is not used at all, then just get rid of it entirely